### ISSUE TITLE
cups_up return 0 when active/total job is zero

### ISSUE THREAD

**Issue by arcenik** [ThoreKr/cups_exporter#6]
On a newly installed cups daemon the exporter fails to export *cups_print_jobs_active* and *cups_print_jobs_total* and *cups_up* is returning 0 despite it show the printers list.

Printer something on a printer fixes the problem. But if you delete that test printer the problem returns.

Here is the exporter output

```
...
# HELP cups_print_jobs_active Number of current print jobs
# TYPE cups_print_jobs_active gauge
# HELP cups_print_jobs_total Total number of print jobs
# TYPE cups_print_jobs_total counter
# HELP cups_printers Number of printers
# TYPE cups_printers gauge
cups_printers 69.0
# HELP cups_printer_status_info Status about printer alerts
# TYPE cups_printer_status_info gauge
cups_printer_status_info{model="Local Raw Printer",printer="bla1",status="happy"} 1.0
cups_printer_status_info{model="Local Raw Printer",printer="bla2",status="happy"} 1.0
cups_printer_status_info{model="Local Raw Printer",printer="bla3",status="happy"} 1.0
...
# HELP cups_up CUPS status
# TYPE cups_up gauge
cups_up 0.0
...
```

**Comment by ThoreKr** [comment]
Sounds like something breaks when trying to gather the job data, as it sequentially sets the printer metrics and afterwards gathers the job data. An except raised here will cause cups_up to be set to 0.

Not sure, if it should wipe all already gathered metrics in this case.

It should print the exception to stdout - can you look that up?
What version of cups are you using?

**Comment by arcenik** [comment]
Hi,

The CUPS version is cups-1.6.3-40.el7.x86_64, on a CentOS7

The stacktrace:
```
Traceback (most recent call last):
  File "./cups_exporter.py", line 139, in <module>
    args.cups_user))
  File "/usr/local/lib/python3.6/site-packages/prometheus_client/registry.py", line 24, in register
    names = self._get_names(collector)
  File "/usr/local/lib/python3.6/site-packages/prometheus_client/registry.py", line 64, in _get_names
    for metric in desc_func():
  File "./cups_exporter.py", line 60, in collect
    self._getJobData()
  File "./cups_exporter.py", line 104, in _getJobData
    lastjobID = list(jobs.keys())[-1]
IndexError: list index out of range
```

And a fix on PR #8
